sdk/spring/spring-cloud-azure-actuator/src/main/java/com/azure/spring/cloud/actuator/storage/StorageBlobHealthIndicator.java

```diff
@@ -35,18 +40,31 @@ public StorageBlobHealthIndicator(BlobServiceAsyncClient blobServiceAsyncClient)
     protected void doHealthCheck(Health.Builder builder) throws Exception {
         if (blobServiceAsyncClient == null) {
             builder.status(NOT_CONFIGURED_STATUS);
-        } else {
+            return;
+        }
+
+        try {
+            BlobContainerAsyncClient containerAsyncClient = blobServiceAsyncClient.getBlobContainerAsyncClient(
+                "spring-cloud-azure-not-existing-container");
+            BlobAsyncClient blobAsyncClient = containerAsyncClient.getBlobAsyncClient(
+                "spring-cloud-azure-not-existing-blob");
```

Comment:
Is it enough for us to construct a `getBlobContainerAsyncClient`, do we really need to construct a `BlobAsyncClient`?